references/cli/provider.go

```diff
@@ -57,12 +59,18 @@ func NewProviderCommand(c common.Args, order string, ioStreams cmdutil.IOStreams
 		},
 	}
 	add, err := prepareProviderAddCommand(c, ioStreams)
-	if err == nil {
+	if err != nil {
+		ioStreams.Errorf("fail to init the provider command:%s \n", err.Error())
```

Comment:
What should the end-users do when hitting this error?

Reply:
This error message is only shown when users execute like `vela provider **` commands. If the cluster cannot communicate, users rectify the fault according to the error message.